lib/cisco_node_utils/radius_server.rb

```diff
@@ -39,6 +39,24 @@ def initialize(name, instantiate=true)
       end
       @name = name
 
+      if platform == :ios_xr
```

Comment:
If the platform is Nexus but these values are provided in the initialize, should we log a warning or anything? Or is ignoring them silently the right thing to do?